types/v56/nsxt_types.go

```diff
@@ -60,11 +79,11 @@ type EdgeGatewayUplinks struct {
 	Dedicated bool `json:"dedicated,omitempty"`
 }
 
-// OpenApiIPRanges is a type alias to reuse the same definitions with appropriate names
-type OpenApiIPRanges = ExternalNetworkV2IPRanges
+// ExternalNetworkV2IPRanges is a type alias to reuse the same definitions with appropriate names
+type ExternalNetworkV2IPRanges = OpenApiIPRanges
 
-// OpenApiIPRangeValues is a type alias to reuse the same definitions with appropriate names
-type OpenApiIPRangeValues = ExternalNetworkV2IPRange
+// ExternalNetworkV2IPRange is a type alias to reuse the same definitions with appropriate names
+type ExternalNetworkV2IPRange = OpenApiIPRangeValues
```

Comment:
Flipping type aliases so that `OpenApiIPRangeValues` is original type, while `ExternalNetworkV2IPRange` is just an alias. The reason for it is that `OpenApiIPRangeValues` is the main type and IDEs automatically suggest main types instead of aliases.

This is non-breaking change.